### DISCUSSION TITLE
Content feed by source

### DISCUSSION THREAD

**Discussion by xNok** [dailydotdev/daily#792] [Feature Requests]
Hi, 

I love the idea of daily.dev, especially the chrome extension and the landing page. The app could do even more for me. What I like to see is a new alternative feed page where I can see filters by sources.

I regularly check dev.to, medium.com and hashnode.com as I want to make sure to get all important content. I would love daily.dev to act more like a personalised aggregator where I can see in one place all the websites I usually consume.

**Comment by idoshamun** [comment]
Hi @xNok,

Can you explain to me how you envision this solution?

**Reply by xNok** [reply]
I see two possible options that could be combined:
* Selecting sources among the one registered (basically filtering by source instead of having the discover
* Let users use daily.dev as an RSS reader; people can register the feeds tey want and have a dedicated page where they can see the latest articles from their preferred sources.

**Comment by UnresolvedCold** [comment]
@xNok, I like this idea of having a personal feed updates along with what is being offered now. It will give daily.dev a personal touch.
